server/app/main.py

```
@@ -0,0 +1,45 @@
+@app.post("/v1/chat/completions")
+async def chat_completions(request: Request):
+    try:
+        body = await request.json()
+        response = litellm.completion(
+            messages=body.get("messages", []),
+            model=body.get("model", "gpt-4o-mini"),
+            temperature=body.get("temperature", 1.0),
+            max_tokens=body.get("max_tokens", None),
+            stream=body.get("stream", False)
+        )
+        logger.info(f"Successfully processed request for model: {model}")
```

Ruff

Undefined name `model`

(F821)